.eslintrc.json

```diff
@@ -105,9 +105,13 @@
                 }
             }
         ],
-
-        // Todo: For each of these, investigate whether we want to enable them ✨
-        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": [
+            "error",
+            {
+                "argsIgnorePattern": "^_([^_]|_*$)",
```

Comment:
For clarity, I do wonder if we should just have `^(_+$|_[^_])` for these, i.e. "all underscores, or starts with only one underscore".

Reply:
makes sense!